lint.sh

```diff
@@ -17,7 +17,7 @@ install(){
     virtualenv $venv
   fi
   . $venv/bin/activate
-  pip install -c constraints.txt -r test-requirements.txt >/dev/null
+  pip install -c constraints.txt -r test-requirements.txt -r requirements.txt >/dev/null
```

Comment:
@hughsaunders I think we determined yesterday that this isn't needed by the gates.  Can you please remove this change?

Reply:
Done